### PR TITLE
Fix SHAs reported in commit messages generated by update-testsuite.sh

### PR DESCRIPTION
**Pull request by sbc100**
Turns out these have been broken for a long time now and don't correspond to actual
commits in the upstream repos.

Also include full URLs so that they are easily clickable in
the web UI.

### CODE REVIEW

File: update-testsuite.sh [hunk 1]
```diff
@@ -117,13 +117,16 @@ for repo in ${repos}; do
     if [ $(git status -s ${wast_dir}/*.wast | wc -l) -ne 0 ]; then
         log_and_run git add ${wast_dir}/*.wast
 
-        repo_sha=$(git -C repos/${repo} log --max-count=1 --pretty=oneline | sed -e 's/ .*//')
-        echo "  ${repo}: ${repo_sha}" >> commit_message
+        repo_sha=$(git -C repos/${repo} log --max-count=1 --oneline origin/master| sed -e 's/ .*//')
```
Comment (abrown) [inline update-testsuite.sh]: So addition of `origin/master` will give use the real commit, not the merge commit?

**Comment by sbc100** [comment]
See https://github.com/WebAssembly/testsuite/pull/14 for an example of the output.

File: update-testsuite.sh (hunk 1 above)
Comment (sbc100) [reply update-testsuite.sh]: Yup.

**Comment by sbc100** [comment]
ping @sunfishcode

**Pull request merged**